src/UI/InputField/InputField.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../styles/variables.scss';
+
+.input-group-UI {
+  position: relative;
+  margin-top: 20px;
+  .eye-icon {
+    position: absolute;
+    top: 21%;
+    right: 0;
+    margin-right: 8px;
+  }
+}
+
+.input-field-UI {
+  border: 2px solid $text-color-middle;
+  border-radius: $main-border-radius;
+  width: 100%;
+  height: 45px;
+  /* flex-grow: 1; */
+  padding: 10px;
+  font-size: 18px;
+}
+
+.input-field-UI.error {
+  border-color: $allert-color-red;
+}
+
+.input-field-UI:hover {
+  border-color: $shaded-mid-light-blue;
+}
+
+.input-field-UI:hover.error {
+  border-color: $allert-color-red;
+}
+
+.input-field-UI:focus {
+  border-color: $shaded-mid-light-blue;
+}
+
+.input-field-UI:focus.error {
+  border-color: $allert-color-red;
+}
+
+.placeholder-label {
+  position: absolute;
+  top: 14px;
+  left: 10px;
+  font-size: 15px;
+  color: $text-color-middle;
+  transition: 0.3s;
+  pointer-events: none;
+}
+
+.placeholder-label.error {
+  color: red;
+}
+
+.input-field-UI:focus + .placeholder-label,
+.input-field-UI:not(:placeholder-shown).input-field-UI:not(:focus)
+  + .placeholder-label {
+  top: -5px;
+  background-color: #fff;
+  font-size: 10px;
+}
+
+.input-error-hidden {
+  visibility: hidden;
+}
+
+.input-error-show {
+  visibility: visible;
+  color: red;
+}
+
+// @media (max-width: 1130px) {
+//   .input-field-UI {
+//     width: 100%;
+//     flex-grow: 1;
+//     padding: 10px;
+//   }
+// }
```

Comment:
Same thing about commented code, as in prev message. All commented code should be deleted or marked why this part of code should be commented.